src/components/StyleGuide/Typography.mdx

```diff
@@ -141,58 +485,117 @@ used for body copy, captions and secondary captions.
 </Box>
 }
 
-### Text Display Sizes
+#### Caption
 
-The `size` prop can be used to render a specific style from the DS default text styles.
+{
 
-- [Typography: Body Styles](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10975%3A52)
-- [Typography: Other Styles](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10975%3A100)
+<List type="ul">
+  <li>Captions below an elements (ex. image, video player, audio player)</li>
```

Comment:
But in the image example, why not use the caption prop for the `Image` component? This example now separates that into two components which I don't think we should recommend.